ext/reflection/php_reflection.c

```diff
@@ -5814,6 +5829,72 @@ ZEND_METHOD(ReflectionProperty, setValue)
 }
 /* }}} */
 
+ZEND_METHOD(ReflectionProperty, getRawValue)
+{
+	reflection_object *intern;
+	property_reference *ref;
+	zval *object;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o", &object) == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	GET_REFLECTION_OBJECT_PTR(ref);
+
+	if (prop_get_flags(ref) & ZEND_ACC_STATIC) {
+		_DO_THROW("May not use getRawValue on static properties");
+		RETURN_THROWS();
+	}
+
+	if (!instanceof_function(Z_OBJCE_P(object), intern->ce)) {
```

Comment:
This is probably my forgetting something, but `object` is not initialized and will remain uninitialized if zero arguments are passed to this function right?